claragenomics/dl4atac/models/models.py

```diff
@@ -7,15 +7,26 @@
 # distribution of this software and related documentation without an express
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
-
+"""Model definitions."""
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
 from claragenomics.dl4atac.layers import ZeroSamePad1d, Activation, ConvAct1d, ResBlock, DownBlock, UpBlock
 
 
 class FC3(nn.Module):
+    """Model with 3 fully connected layers."""
```

Comment:
@avantikalal for this whole file to update the model descriptions

Reply:
@ntadimeti can you delete the FC2 and FC3 models, they were just there for experimentation and are never used.